src/Api/Controllers/GroupsController.cs

```diff
@@ -133,7 +137,27 @@ public async Task Delete(string orgId, string id)
             throw new NotFoundException();
         }
 
-        await _groupService.DeleteAsync(group);
+        await _deleteGroupCommand.DeleteAsync(group);
+    }
+
+    [HttpDelete("")]
+    [HttpPost("delete")]
```

Comment:
We previously only had an endpoint to delete a single group at a time. This is adding the bulk delete endpoint accepting a list of group Ids in the request body (instead of a single group Id in the url)